snorkel/labeling/analysis.py

```diff
@@ -323,9 +325,11 @@ def lf_summary(
             Summary statistics for each LF
         """
         n, m = self.L.shape
-        if lf_names is not None:
+        lf_names: Union[List[str], List[int]]
+        if self._lf_names is not None:
             col_names = ["j"]
```

Comment:
col_names seems to just be the keys of d. So we can have d be a collection.OrderedDict and just d.keys() in place of col_names.